pysc2/pysc2/lib/actions.js

```diff
@@ -136,75 +137,90 @@ function raw_no_op(/*action*/) {
 
 function raw_move_camera(action, world) {
   // Move the camera.//
-  const action_cmd = action.action_raw.camera_move
+  const action_cmd = action.getActionRaw().getCameraMove()
   world.assign_to(action_cmd.center_world_space)
 }
 
 function raw_cmd(action, ability_id, queued, unit_tags) {
   // Do a raw command to another unit.//
-  const action_cmd = action.action_raw.unit_command
-  action_cmd.ability_id = ability_id
-  action_cmd.queue_command = queued
-  if (!isinstance(unit_tags, [Array])) {
-    unit_tags = [unit_tags]
+  const action_cmd = action.getActionRaw().getUnitCommand()
+  action_cmd.setAbilityId(ability_id)
+  action_cmd.setQueueCommand(queued)
+  if (isinstance(unit_tags, [Array])) {
+    unit_tags.forEach((unit_tag) => {
+      action_cmd.addUnitTags(unit_tag)
+    })
+  } else {
+    action_cmd.addUnitTags(unit_tags)
```

Comment:
If unit_tags is not array, then do we need to change it to array?

Reply:
no, look at what the code does
if array 
  add each tag elementwise 
else
 add single tag

Reply:
In the python code they just wanted to make use of List.extends